src/main/java/gov/nasa/jpl/mbee/mdk/api/incubating/MDKConstants.java

```diff
@@ -46,5 +46,6 @@
             TWC_VERSION_KEY = "twcVersion",
             ARTIFACT_IDS_KEY = "_artifactIds",
             CHECKSUM_KEY = "checksum",
-            CONTENT_TYPE_KEY = "contentType";
+            CONTENT_TYPE_KEY = "contentType",
+            SYSML_PROFILE_LOCAL_ID = "_11_5EAPbeta_be00301_1147434586638_637562_1900";
```

Comment:
Rename to SYSML_PROFILE_ID. That way we leave implementation detail out of variable names and can swap out without refactoring. The ID system is especially prone to change.